target/linux/kirkwood/base-files/etc/diag.sh

```diff
@@ -14,6 +14,9 @@ get_status_led() {
 	pogo_e02)
 		status_led="pogo_e02:orange:fault"
 		;;
+	pogo_v4)
+		status_led="pogoplugv4:red:fault"
```

Comment:
Use the `pogoplugv4:green:health` LED here. Using a red (warning/error) LED by default doesn't seem to me a good idea.